server/src/kraftfile/values/targets.ts

```diff
@@ -0,0 +1,217 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+import {
+    CompletionItemKind,
+    CompletionItem,
+} from 'vscode-languageserver/node'
+import { unikraft } from '../../utils'
+import { codeBlockStr, getInLineAttribute } from '../utils';
+
+export function targetsValueCompletionItem(paretnAttribute: string): CompletionItem[] {
+    if (paretnAttribute !== "targets") {
+        return [];
+    }
+
+    const qemu86 = "qemu/x86_64"
+    const qemu64 = "qemu/arm64"
+    const fc86 = "firecracker/x86_64"
+    const fc64 = "firecracker/arm64"
+    const xen86 = "xen/x86_64"
+    const xen64 = "xen/arm64"
```

Comment:
these variable names are not very suggestive, it's more like x86 vs arm, not 86 vs 64 😅

Reply:
Done 👍🏽